servers/zts/src/main/java/com/yahoo/athenz/zts/ZTSImpl.java

```diff
@@ -2475,8 +2485,14 @@ boolean verifyAWSAssumeRole(String domainName, String roleResource, String princ
         return false;
     }
 
-    X509CertRecord insertX509CertRecord(ResourceContext ctx, final String cn, final String provider,
-            final String instanceId, final String serial, Boolean certUsage) {
+    X509CertRecord insertX509CertRecord(ResourceContext ctx,
```

Comment:
Added four new columns but only two of them, expiryTime and hostName can be inserted. The other two, lastNotifiedTime and lastNotifiedServer are updated as part of the Notification logic.